website/docs/guides/3.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,1053 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: 3.0 Upgrade Guide"
+description: |-
+Azure Resource Manager: 3.0 Upgrade Guide
+
+---
+
+# v3.0 of the AzureRM Provider
+
+The last major release for the AzureRM provider was in February 2020.
+
+Since then we've added support for nearly 400 Resources and 100 Data Sources, bringing the total supported features to 751 Resources and 228 Data Sources at the time of writing this guide.
+
+Version 3.0 of the AzureRM Provider is a major release and as such includes some larger-scale changes which are outlined in this document.
+
+!> **NOTE:** Version 3.0 of the AzureRM Provider supports Terraform 0.12 and later - Terraform 0.10 & 0.11 are no longer supported with this release.
+
```

Comment:
Should we add instructions here for how to opt-in and opt-out of the Beta and how to leave feedback?
What we did for [app service beta](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/3.0-app-service-beta#opting-into-the-app-service-beta-resources) is a good example.
Or we could make a page like that for 3.0 Beta in the registry docs and then link it here in the upgrade guide?

Reply:
Added :)